draft-ietf-rtcweb-jsep.xml

```diff
@@ -4076,6 +4076,65 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             process, as a remote offer or answer, unmodified SDP coming
             from any other implementation of this specification.</t>
         </section>
+        <section title="Processing RTP packets"
```

Comment:
This text needs to clearly say that the SRTP has been validated before this algorithm is executed, because it's otherwise not safe to update the SSRC table based on anything you see in packets.